nautobot/docs/user-guide/core-data-model/extras/configcontext.md

```diff
@@ -2,8 +2,8 @@
 
 Sometimes it is desirable to associate additional data with a group of devices or virtual machines to aid in automated configuration. For example, you might want to associate a set of syslog servers for all devices within a particular region. Context data enables the association of extra user-defined data with devices and virtual machines grouped by one or more of the following assignments:
 
-* Region
-* Site
+* Location
+* LocationType
```

Comment:
Double-check me on this but I don't think you can scope a ConfigContext by LocationType, only by Location.